buildout.cfg

```diff
@@ -31,3 +36,6 @@ domain = zope.errorview
 output = src/zope/errorview/i18n/locales
 zcml =
   <include package="zope.i18n" file="meta.zcml" />
+
+[sources]
+z3c.recipe.i18n = git ${buildout:github}/z3c.recipe.i18n pushurl=${buildout:github_push}/z3c.recipe.i18n branch=py3
```

Comment:
You plan to remove these once a Python-3-capable z3c.recipe.i18n is released to PyPI?